### PR TITLE
React state with Mats

### PR DESCRIPTION
**Pull request by mats-johansen**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,110 @@
...
+
+button:hover {
+  opacity: 0.7;
+}
+
+form * {
+  font-size: inherit;
+  font-weight: inherit;
+  font-family: inherit;
+}
```
Comment (petrame) [inline code/src/index.css:110]: Well structured css code and easy to read.

**Review by petrame — APPROVED** [review]
Hello Mats! Great job with the survey!
I had no problem reading the code, although some comments here and there for future you wouldn't hurt :)
I think the design is cool. The font, image and button fit well together.
Also, great job with the required attribute, I know many of us struggled with that.

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,80 @@
...
+
+  const surfboards = ['Shortboard', 'Groveler surfboard', 'Fish surfboard', 'Funboard', 'Longboard', 'Big-wave board'];
+  const waves = ['Weak', 'Medium', 'Powerful'];
+  const numberOfQuestions = 3;
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setSection(section + 1)
+    section === numberOfQuestions && setShowSummary(true)
+  }
```
Comment (HIPPIEKICK) [inline code/src/App.js:25]: semicolons

File: code/src/InputSelect.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <option value ="" disabled>Select surfboard</option>
+
+       {array.map(item => (
+        < option key={item} value={item} >{item} </option>
+        ))}
+
+      </select>
+    </div>
+  )
+}
```
Comment (HIPPIEKICK) [inline code/src/InputSelect.js:24]: semicolons

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Take a stand on whether you're using semicolons or not - either everywhere or not at all :) Changing to CR until it's consistent. Also - I saw that you still have issues with VSC formatting (as in `InputRadio.js`). Did you get any clarity on that?

File: README.md [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
+Had some issues understandning state and styling the forms & radio-buttons in a nice looking way. But progress is being made!
```
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply README.md:6]: Agree on the nice design!

**Comment by mats-johansen** [comment]
> Take a stand on whether you're using semicolons or not - either everywhere or not at all :) Changing to CR until it's consistent. Also - I saw that you still have issues with VSC formatting (as in `InputRadio.js`). Did you get any clarity on that?

Oh noticed that I was inconsistent on the semicolons now! No still working on the formatting issue, still have to change between javascript and javascript react when saving. Hopefully it will start working after the next troubleshoot session haha

**Comment by HIPPIEKICK** [comment]
> > Take a stand on whether you're using semicolons or not - either everywhere or not at all :) Changing to CR until it's consistent. Also - I saw that you still have issues with VSC formatting (as in `InputRadio.js`). Did you get any clarity on that?
> 
> Oh noticed that I was inconsistent on the semicolons now! No still working on the formatting issue, still have to change between javascript and javascript react when saving. Hopefully it will start working after the next troubleshoot session haha

Did you try to update / reinstall VSC? Uninstalling formatters (like prettier)?

**Review by HIPPIEKICK — APPROVED** [review]

**Comment by HIPPIEKICK** [comment]
Regarding formatting issues, take a look at this: https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code

> Note: If you do not see a prompt for selecting a default format, you can manually change this in your Settings. Set Editor: Default Formatter to ebsenp.prettier-vscode.

**Pull request closed without merging**